x-pack/plugins/ml/public/application/components/anomalies_table/anomalies_table_columns.js

```diff
@@ -110,9 +110,21 @@ export function getColumns(
     {
       field: 'severity',
       'data-test-subj': 'mlAnomaliesListColumnSeverity',
-      name: i18n.translate('xpack.ml.anomaliesTable.severityColumnName', {
-        defaultMessage: 'severity',
-      }),
+      name: (
+        <EuiToolTip
+          content={i18n.translate('xpack.ml.overview.anomalyDetection.tableSeverityTooltip', {
+            defaultMessage:
+              'A normalized score between 0-100, which indicates the relative significance of each anomaly',
```

Comment:
```suggestion
              'A normalized score between 0-100, which indicates the relative significance of each anomaly.',
```

Reply:
To help address some of the confusion around the scores at different levels of our results (bucket, influencer, record), could we get the word `record` in here somewhere, to show that the table is displaying record scores?

Reply:
Good point, I've added that text in 597391d5727b66309ac3a1dfe017810b6840162c